apps/finance/app/src/components/ToggleFiltersButton.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from 'styled-components'
+import { theme } from '@aragon/ui'
+
+const StyledButton = styled.button`
+  border: none;
+  background: none;
+  height: 16px;
+  width: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  cursor: pointer;
+  outline: none;
+
+  &:focus {
+    border: 2px solid ${theme.accent};
```

Comment:
We should use `outline` here so that it doesn’t impact the box:

<img width="44" alt="image" src="https://user-images.githubusercontent.com/36158/51471850-1eb57400-1d78-11e9-9526-47a0dc4ef30e.png">

And also remove the inner border in Firefox:

```css
&::-moz-focus-inner {
  border: 0;
}
```